pkg/repstream/streampb/stream.proto

```diff
@@ -75,6 +75,7 @@ message ReplicationProducerRequest {
     (gogoproto.nullable) = false,
     (gogoproto.customname) = "TenantID"
   ];
+  bool cutover_retention = 4;
```

Comment:
I don't think we need this bool. The lack of a destination cluster id should be enough for us to know this.

Reply:
Done.